src/components/TabsContent.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { ListingItem } from "./ListingItem";
+import { useSelector } from "react-redux";
+import { ApplicationState } from "../app/store";
+import { ListItem } from "../types/ListItem";
+import { Tab } from "../types/Tab";
+
+export type TabsContentProps = {
+  displayTabs: Tab[];
+  selectedTabIndex: number;
+};
+export const TabsContent: (props: TabsContentProps) => JSX.Element = ({
+  displayTabs,
+  selectedTabIndex,
+}: TabsContentProps) => {
+  const { items } = useSelector((state: ApplicationState) => state);
+  return (
+    <div>
+      {displayTabs.map((tab) => {
+        let itemsToDisplay: ListItem[] = [];
+        if (tab.title === "to read") {
+          itemsToDisplay = items.filter((item) => !item.alreadyRead);
+        } else if (tab.title === "already read") {
```

Comment:
I believe that the `"already read"` and `"to read"` constants repeat throughout the app and imo should be extracted to a reused/able constant.

Reply:
Great! Totally true.